docs_devel/docs/40.ContributingDocument.md

```diff
@@ -2,9 +2,26 @@
 
 OmegaT Manuals and User guides are co-developed on OmegaT program repository.
 
-You can see built html pages on `docs/` folder and its source is located at `doc_src/` folder in OmegaT project
-source code repository.
+You can see a document source in`doc_src/` folder in OmegaT project source code repository.
 
 ## Documentation rules and procedures
 
-(TBD)
+Please contact to [Documentation manager](https://omegat.org/about) at [omegat-development mailing list]
+(https://sourceforge.net/projects/omegat/lists/omegat-development) for details.
```

Comment:
```suggestion
Please contact to [Documentation manager](https://omegat.org/about) at [omegat-development mailing list](https://sourceforge.net/projects/omegat/lists/omegat-development) for details.
```

Reply:
Why would people need to contact the documentation manager ? People can do as they do for the code. Send PRs after creating issues on SF, on the documentation tracker.

Reply:
Because I cannot find any document to describe rules and procedure in omegat.org website, and also in source tree. Please give a description. Otherwise, I can revert it to previous sentence; "[TBD]".

Reply:
This may be valuable to add from https://omegat.org/support

> documentation issues are handled on SourceForge by the [Documentation tracking system](https://sourceforge.net/p/omegat/documentation/). Please do not use these tracking systems before first raising the issue on the user group and/or mailing lists.